scripts/deploy-gateway.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+require('dotenv').config();
```

Comment:
Do you not need to install dotenv?

Reply:
Yes, because there are still a few parameters loaded from there such as the rpc url and the account private key. Unless you think its preferable to pass them as arguments?